packages/arb-token-bridge-ui/src/token-bridge-sdk/Erc20WithdrawalStarter.ts

```diff
@@ -86,6 +86,15 @@ export class Erc20WithdrawalStarter extends BridgeTransferStarter {
     // no-op
   }
 
+  /**
+   * Most tokens inherently allows the token gateway to burn whichever amount
+   * on the child chain for withdrawal because they inherited the
+   * IArbToken interface that allows the gateway to burn without allowance approval
```

Comment:
```suggestion
   * Most tokens inherently allow the token gateway to burn the withdrawal amount
   * on the child chain because they inherit the
   * IArbToken interface that allows the gateway to burn without allowance approval
```